reference/6/PSDiagnostics/Get-LogProperties.md

```diff
@@ -1,42 +1,55 @@
 ---
 external help file: PSDiagnostics-help.xml
-keywords: powershell,cmdlet
-locale: en-us
 Module Name: PSDiagnostics
-ms.date: 06/09/2017
+online version:
 schema: 2.0.0
+ms.date:  11/27/2018
+Help Version: 6.1.0.2
 ---
 
 # Get-LogProperties
 
 ## SYNOPSIS
-{{Fill in the Synopsis}}
+Retrieves the properties of a Windows event log.
 
 ## SYNTAX
 
 ```
-Get-LogProperties [-Name] <Object> [<CommonParameters>]
+Get-LogProperties [-Name] <string> [<CommonParameters>]
 ```
 
 ## DESCRIPTION
-{{Fill in the Description}}
+
+This cmdlet gets the configuration settings of a Windows event log. This cmdlet is used by the
+`Enable-PSTrace` and `Disable-PSTrace` cmdlets.
+
+This cmdlet only applies to computers running Windows.
```

Comment:
This description isn't in all the updated cmdlet docs.  Is this true only of this specific cmdlet, but the others (at least in 6) work cross platform?

Reply:
I copied these files to all versions. They should be the same. I will check.

Reply:
We'll mark this as resolved, but please open an Issue to ensure it gets done at a later date.